Examples/SmokeTest/SmokeTest/UIKitView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UIKView.swift
+//  SmokeTest
+//
+//  Created by Arri Blais on 11/13/24.
+//
```

Comment:
You should delete these headers. You can change the template in Xcode settings somewhere.